src/chrome/content/rules/PHP.xml

```diff
@@ -65,14 +67,26 @@
 	<target host="*.php.net" />
 
 
-	<!--	Not secured by server:
-					-->
-	<!--securecookie host="^bugs\.php\.net$" name="^PHPSESSID$" /-->
-
+	<securecookie host="^bugs\.php\.net$" name="^PHPSESSID$" />
 	<securecookie host="^(?:\w*\.)?php\.net$" name=".+" />
 
 
-	<rule from="^http://((?:bugs|downloads|edit|git|master|museum|pear|qa|shared|snaps|static|svn|wiki|www)\.)?php\.net/"
+	<rule from="^http://((?:bugs|downloads|edit|git|master|museum|pear|qa|shared|snaps|static|svn|wiki)\.)?php\.net/"
 		to="https://$1php.net/" />
 
+  <test url="http://php.net/" />
+  <test url="http://bugs.php.net/" />
```

Comment:
You can simplify this ruleset by moving these to `<target>`s, e.g.

```
<target host="bugs.php.net" />
<target host="downloads.php.net" />
```

Then you can replace the rule with:

```
<rule from="^http:"
        to="https:" />
```

And you won't need the test URLs because they are autogenerated by the target tags.

This is optional: If you'd like to submit as-is, that's fine. The changes I suggest will just make maintenance slightly simpler in future.